GRADING.md

```diff
@@ -0,0 +1,46 @@
+| 3         | Adequate organisation | Adequate organisation with an introduction, main points, and conclusion, but transitions are lacking. Some clarity in the presentation, with minimal technical detail. |
+| 4         | Good organisation     | Mostly logical flow with a clear introduction, main points, and conclusion. Some transitions may be abrupt. Organised presentation with sufficient technical detail.      |
+| 5         | Thoroughly organised  | Concise and well-organised presentation with a clear introduction, main points, and conclusion. Smooth transitions between topics. Excellent depth of technical detail.   |
+
+### Tips 
+- Assess how well the student comprehends the technical project. Look for evidence of in-depth research and knowledge.
+- Evaluate the clarity and coherence of the information presented. Check if complex concepts are explained clearly and concisely.
+- Look for a logical progression of ideas and a well-structured sequence of topics. Assess the transitions between sections.
+
+## Delivery and Engagement
+
+This section focuses on the student's manner of presenting, including engagement, confidence, pace of speech, and overall ability to keep the audience interested and attentive. It measures the overall impression the presentation leaves on the audience, including the student's ability to captivate, inspire interest, and create a memorable and impactful experience for the listeners.                 
+
+| **Score** | **Caption**                            | **Details**                                                                                                               |
+| --------- | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| 0         | Did not present                        | Student did not present.                                                                                                  |
+| 1         | Ineffective delivery                   | Fails to engage the audience effectively, lacks pace, constant distractions. Navigates challenges or hiccups poorly       |
+| 2         | Delivery is awkward or distracting     | Limited audience engagement, and inconsistent pace.                                                                       |
+| 3         | Adequate delivery                      | Adequate engagement with the audience, reasonable pacing with only one or two distractions. Limited challenges or hiccups |
+| 4         | Mostly engaging and confident delivery | Maintains a reasonable pace. Mostly captivates the audience, with some elements fostering interest and engagement.        |
+| 5         | Engaging and confident delivery        | Appropriate pace, and varied tone. Captivates the audience, fostering interest in a smooth impactful presentation.        |
```

Comment:
Removed references to audience engagement as that's hard to measure and not entirely reflective of the student's presentation skills

```suggestion
| **Score** | **Caption** | **Details**                                                                                                                                           |
| --------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
| 0         | n/a         | Student did not present.                                                                                                                              |
| 1         | Poor        | Very low confidence, appears unprepared/unrehearsed. Haphazard pacing, encounters major issues or navigates issues poorly.                            |
| 2         | Fair        | Limited confidence and inconsistent pacing. Shows some level of understanding with only a few stumbles, but could be better prepared.                 |
| 3         | Adequate    | Decent pacing with only one or two distractions. Reasonably confident presentation, but doesn't encounter any major issues.                           |
| 4         | Good        | Maintains a reasonable pace. Mostly confident delivery, with the ability to clearly communicate the process and outcomes of the project.              |
| 5         | Excellent   | Presents with consistent pacing and engaging tone. Appears confident and knowledgeable throughout, displays a clear level of passion for the project. |
```

Reply:
Will integrate this into a commit. Some wording I'll change slightly. E.g. the change to point 3 sounds a little like we'd mark them down for _not_ having encountered issues, rather than an acknowledgment that there are less issues than someone who achieves a lower score.